hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2753,7 +2753,7 @@
   </property>
   <property>
     <name>ozone.network.topology.aware.read</name>
-    <value>false</value>
+    <value>true</value>
```

Comment:
Default value should also be updated at:

https://github.com/apache/ozone/blob/cce3c7ea79ec19657d10f011ddeaf3067a2f83bd/hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java#L487